M2/Macaulay2/m2/indeterminates.m2

```diff
@@ -90,6 +90,12 @@ runLengthEncode0 = x -> (
 		    (dupin = null; oi = i; m = 1))));
      x)
 
+rle = method(Dispatch => Thing)
```

Comment:
What is this function supposed to do, and why is it needed?

Reply:
It's not something I've added, it used to be there before, I just moved it. I think it's subtly different from `runLengthEncode`.